src/screens/poa-form/styles.scss

```diff
@@ -8,17 +8,22 @@
 
 .error {
   color: #cc0000;
+  display: block;
 }
 
-.grommetux-text-input.grommetux-input.input-class-long {
+.grommetux-text-input.grommetux-input.input-class-long{
   margin: 0em 0.5em 1em 0em;
   width: 100%;
 }
 
-.grommetux-text-input.grommetux-input.input-class {
+.grommetux-text-input.grommetux-input.input-class, .grommetux-select  {
   margin: 0em 1em 1em 0em;
 }
 
+.grommetux-select, .grommetux-select__drop{
```

Comment:
Should any of this be site wide? You could add it to the overrides in `@/styles/grommet/`

Reply:
We only have one form on the site. If we add more forms, I think it would be time to evaluate how we want to handle this. We may not want this global and if we don't we probably need to use more specific css classes. I think its fine until it becomes an issue.